nautobot/extras/api/serializers.py

```diff
@@ -481,6 +482,38 @@ class Meta:
         ]
 
 
+#
+# Scheduled Jobs
+#
+
+
+class ScheduledJobSerializer(serializers.ModelSerializer):
+    url = serializers.HyperlinkedIdentityField(view_name="extras-api:scheduledjob-detail")
+    user = NestedUserSerializer(read_only=True)
+    approved_by_user = NestedUserSerializer(read_only=True)
+
+    class Meta:
+        model = ScheduledJob
+        fields = [
+            "id",
+            "url",
+            "name",
+            "user",
+            "task",
+            "interval",
+            "queue",
+            "job_class",
+            "last_run_at",
+            "total_run_count",
+            "date_changed",
+            "description",
+            "user",
+            "approved_by_user",
+            "approval_required",
+            "approved_at",
+        ]
```

Comment:
Requesting a sanity check here on whether all of these fields are useful/relevant to include at this time - in particular I think that maybe `task` and `queue` are currently always going to be `"nautobot.extras.jobs.scheduled_job_handler"` and `None` respectively?

Reply:
I’m not sure whether they are useful. My thinking was that if in doubt, I’d rather have the field and ignore it than not have it and need it, but I agree that this is a lot of potentially not-so-useful data.